contrib/pyln-client/pyln/client/gossmap.py

```diff
@@ -164,6 +211,15 @@ def __eq__(self, other):
     def __hash__(self):
         return self.scid.__hash__()
 
+    def has_feature(self, bit):
+        return 1 << bit & self.features > 0
+
+    def has_feature_odd(self, bit):
+        return 2 << bit & self.features > 0
+
+    def has_features(self, bitmask):
+        return bitmask & self.features == bitmask
```

Comment:
Shouldn't the equality be a subset test? Otherwise we only get `has_features` when we present it with the full list, not a subset we might be interested in.

Reply:
Using `bitmask & self.features == bitmask` (as in my current code) will allow to filter for a (complete) subset of features using a `bitmask`. Do you prefer a function that returns True if any of the subset `bitmask` bits was returned? I find that not a really helpful function signature when it comes to identifying nodes with a certain compatibility.

E.g. I want to identify all nodes that have mandatory `GOSSIP_QUERY_EX` and mandatory `BASIC_MPP`, I can use one call that returns all of them:

`stats.filter_nodes_features(2 << LnFeatureBits.GOSSIP_QUERIES_EX | 2 << LnFeatureBits.BASIC_MPP)`